core/controllers/admin.py

```diff
@@ -212,19 +212,19 @@ def get(self):
     @acl_decorators.can_access_admin_page
     def post(self):
         """Handles POST requests."""
-        action = self.normalized_payload.get('action')
+        action = self.HANDLER_ARGS_SCHEMAS.get('action')
```

Comment:
Why the changes in this file? They doesn't make sense.

Reply:
This is because normalized payload was causing an error as it did not have the property specified in the statement and when I checked the class, the property was written under handler args schema

Reply:
That is most probably wrong, it doesn't make sense to use `self.HANDLER_ARGS_SCHEMAS`, please set it back to `self.normalized_payload` everywhere and share the error here.